addons/l10n_ch/models/res_bank.py

```diff
@@ -34,6 +34,24 @@ class ResPartnerBank(models.Model):
     _inherit = 'res.partner.bank'
 
     l10n_ch_postal = fields.Char(string='ISR reference', help='The ISR number of the company within the bank')
+    acc_type = fields.Selection(
+        selection=lambda x: x.env['res.partner.bank'].get_supported_account_types(),
+        compute='_compute_acc_type',
+        store=True,
```

Comment:
@hpr-odoo @rgo-odoo Why do we need this to be store True?

Reply:
@jco-odoo cause we had to use `acc_type` in SQL query to create partial index on `res.partner.bank`, see here https://github.com/odoo/odoo/pull/34476/files#diff-b6235a49c5949673bb213f3f712026cbR53